src/Compiler/Checking/AttributeChecking.fs

```diff
@@ -503,15 +503,17 @@ let PropInfoIsUnseen m pinfo =
 #endif
 
 /// Check the attributes on a union case, returning errors and warnings as data.
-let CheckUnionCaseAttributes g (x:UnionCaseRef) m =
-    CheckEntityAttributes g x.TyconRef m ++ (fun () ->
-    CheckFSharpAttributes g x.Attribs m)
+let CheckUnionCaseAttributes g (x:UnionCaseRef) m = trackErrors {
+    do! CheckEntityAttributes g x.TyconRef m
+    do! CheckFSharpAttributes g x.Attribs m
+}
```

Comment:
Could you please re-format it, to have the CE on the new line, it's arguably easier to understand that you're inside specific CE when it's visible on its own line?